safe/utilities/test/test_metadata.py

```diff
@@ -24,6 +24,7 @@
 
 class TestMetadataUtilities(unittest.TestCase):
     """Test for Metadata Utilities module."""
+    maxDiff = None
```

Comment:
Landscape likes when there is one blank line before and after class docstring ;-)